usaspending_api/etl/elasticsearch_loader_helpers/controller_for_spark.py

```diff
@@ -64,8 +64,6 @@ def ensure_view_exists(self, sql_view_name: str, force_recreate=True) -> None:
             # Replace the Postgres regex operator with the Databricks regex operator
             identifier_replacements["~"] = "rlike"
             identifier_replacements["state_data"] = "global_temp.state_data"
-            identifier_replacements["array_agg"] = "collect_list"
-            identifier_replacements["json_agg"] = "collect_list"
```

Comment:
Any reason to remove these? Could still be good to have these replacements in place in cast we need to use `array_agg` or `json_agg` in the future

Reply:
They were only removed, because `ARRAY_AGG` and `JSON_AGG` are no longer present in the SQL and don't need to be replaced anymore. I removed them to be consistent with the other `load_type` keys and only replace strings that are currently present in the SQL.

Doesn't bother me to add these back though for future reference, if nothing else.

Reply:
These were added back.